### PR TITLE
GSSOC'24 Issue #979 - Adding PDF Utility Tool to Python in Basics in …

### LINKED ISSUE #979: Title: GSSOC'24 - Desktop Application - Python TKinter - PDF Utility tool app
### Is your feature request related to a problem? Please describe.

The Desktop Application will allow users to merge pdfs, convert .docx to pdf, split pdf and password protect the pdf. Since it will be a desktop application, it won't require any internet connection or sharing your pdf files over the cloud which may not be as secure. 

### Describe the solution you'd like.

image

### Describe alternatives you've considered.

Alternatives can be adding more features to the PDF Utility desktop application

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by KJ173**
…Desktop Applications

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

I have added 'PDF Utility Tool' in Python under Basics in Desktop Applications.
This tool allows 4 functions:
-> Merge Selected PDFs and save it in desired location
-> Convert a PDF to .docx file
-> Password protect a pdf file
-> Split pdf based on page number into 2 pdfs
This helps to perform PDF manipulations offline for free, instead of uploading sensitive documents on the cloud on random websites.
image

Fixes #979 

### Checklist

- [ yes ] I've read the contribution guidelines.
- [ yes ] I've checked the issue list before deciding what to submit.
- [ yes ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
GSSOC'24 Issue #979

### CODE REVIEW

File: Desktop Application/Basic/Python/PDF Utility Tool/readme.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+The PDF Toolkit uses the following Python packages:
+
+* Pillow (PIL): Image processing library used for handling images.
+* PyPDF2: Library for working with PDF files, providing functionalities for merging, splitting, and encrypting PDFs.
+* pdf2docx: Converter library used to convert PDF files to DOCX format.
+
+## GUI
+
+![1715583384233](image/readme/1715583384233.png "GUI Interface")
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/PDF Utility Tool/readme.md:38]: Image is not working.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@KJ173 You have to add good `readme.md` file and make sure your image should work

**Comment by Kushal997-das** [comment]
@KJ173 Update on this ?

**Comment by KJ173** [comment]
> @KJ173 Update on this ?

Will be updating it by tomorrow. Till then please assign labels as gssoc and level as well.
thank you!

**Comment by KJ173** [comment]
@Kushal997-das Hi I have made the required changes. Please add tags and accept the pull request.
Thank you!

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@KJ173 Kindly add your project under python basic [here](https://github.com/Kushal997-das/Project-Guidance/blob/main/Desktop%20Application/README.md)

**Comment by KJ173** [comment]
> @KJ173 Kindly add your project under python basic [here](https://github.com/Kushal997-das/Project-Guidance/blob/main/Desktop%20Application/README.md)

Have added the project in this index. Kindly review and merge the commits. 
Thank you!

**Review by Kushal997-das — APPROVED** [review]
@KJ173 💯

**Pull request merged**